imguiWrapper.cpp

```diff
@@ -407,3 +407,8 @@ IggBool iggIsItemHovered(int flags)
 {
    return ImGui::IsItemHovered(flags) ? 1 : 0;
 }
+
+IggBool iggListBox(char const *label, int* current_item, char const* const items[], int items_count, int height_items)
+{
+    return ImGui::ListBox(label, current_item, items, items_count, height_items);
```

Comment:
Here is an implicit conversion from `bool` to `IggBool`. While this works out, I prefer to have this explicit, as with the others. Please add `...) ? 1 : 0;`.